packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -226,3 +222,13 @@ module.exports = {
     tls: 'empty'
   }
 };
+
+// @remove-on-eject-begin
+// Override babel-loader options to not read .babelrc.
+var babelLoader = module.exports.module.loaders.find(l => l.loader === 'babel');
+babelLoader.query = babelLoader.query || {};
+babelLoader.query.babelrc = false;
+// We embed the preset instead of referring to it because otherwise
+// `env` options in the preset do not get taken into account.
```

Comment:
Figured it out, it’s a Babel bug: https://github.com/babel/babel/issues/4539. For now working around it in the preset. To be honest I don’t remember why we use `require.resolve()` 😄 . Maybe we don’t even need it.